File: src/main/java/org/goafabric/core/data/controller/vo/Organization.java
package org.goafabric.core.data.controller.vo;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Null;
import jakarta.validation.constraints.Size;

import java.util.List;

public record Organization(
    @Null String id,
    @NotNull @Size(min = 3, max = 255) String name,

    List<Address> address,
    List<ContactPoint> contactPoint
) {}

File: src/main/kotlin/org/goafabric/core/organization/controller/dto/Organization.kt
package org.goafabric.core.organization.controller.dto

data class Organization(
    var id: String? = null,
    var version: Long? = null,
    var name: String? = null,
    var bsnr: String? = null,
    var address: MutableList<Address> = mutableListOf(),
    var contactPoint: MutableList<ContactPoint> = mutableListOf()
)
